cmd/bbolt/main.go

```diff
@@ -60,12 +60,25 @@ func main() {
 	m := NewMain()
 	if err := m.Run(os.Args[1:]...); err == ErrUsage {
 		os.Exit(2)
+	} else if err == ErrUnknownCommand {
+		execute()
 	} else if err != nil {
 		fmt.Println(err.Error())
 		os.Exit(1)
 	}
 }
 
+func execute() {
+	rootCmd := NewRootCommand()
+	if err := rootCmd.Execute(); err != nil {
+		if rootCmd.SilenceUsage {
+			exitWithError(1, err)
```

Comment:
The contents of the function can be inlined. It's only 2 lines of code but requires jumping to another file to see contents.

Reply:
It seems not a big deal, but updated per your comment anyway.

Reply:
Thanks